MonopolyKata/Monopoly.cs

```diff
@@ -51,7 +59,7 @@ private void DetermineOrder()
             Random order = new Random();
             RollOrder[0] = order.Next(1, 4);
             RollOrder[1] = order.Next(1, 4);
-            while(RollOrder[1] == RollOrder[0])
+            while (RollOrder[1] == RollOrder[0])
                 RollOrder[1] = order.Next(1, 4);
```

Comment:
What is this block of code doing? Im not sure i understand the purpose of the while loop.